trustymail/domain.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, domain_name, timeout, smtp_timeout, smtp_localhost, smtp_port
         # 3. Whether or not the server supports STARTTLS
         self.starttls_results = {}
 
-        # A list of any errors that occurred while scanning records.
-        self.errors = []
+        # A list of any debugging information collected while scanning records.
+        self.debug = []
```

Comment:
I'm a bit leery of the name `.debug` as that makes me think of a boolean flag more than a list... maybe `.debug_info` would be more descriptive?

Reply:
I'll make a separate issue for this.  The problem is that changing this name requires a pull request for `domain-scan` as well as one for our `trustymail` report-generation code.  Since the `trustymail` reports will be autogenerated this weekend there isn't time to test and get those other pull requests in.

Reply:
OK, I'll make the change.